### PR TITLE
ktlint 1.0

### PR DESCRIPTION
**Pull request by JakeWharton**
and its terrible function naming rule that doesn't honor Compose conventions!

Closes #322

### CODE REVIEW

File: molecule-gradle-plugin/src/main/java/app/cash/molecule/gradle/MoleculePlugin.kt [hunk 1]
```diff
@@ -36,8 +36,8 @@ import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet.Companion.COMMON_MAIN_
 import org.jetbrains.kotlin.gradle.plugin.SubpluginArtifact
 import org.jetbrains.kotlin.gradle.plugin.SubpluginOption
 
-private const val extensionName = "molecule"
-private const val moleculeRuntime = "app.cash.molecule:molecule-runtime:$moleculeVersion"
+private const val EXTENSION_NAME = "molecule"
```
Comment (mattprecious) [inline molecule-gradle-plugin/src/main/java/app/cash/molecule/gradle/MoleculePlugin.kt:39]: Java called...

**Review by mattprecious — APPROVED** [review]

File: molecule-gradle-plugin/src/main/java/app/cash/molecule/gradle/MoleculePlugin.kt (hunk 1 above)
Comment (JakeWharton) [reply molecule-gradle-plugin/src/main/java/app/cash/molecule/gradle/MoleculePlugin.kt:39]: Yeah I hate it, but https://kotlinlang.org/docs/coding-conventions.html#property-names

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -41,7 +41,7 @@ kotlinx-coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-t
 
 kotlinx-binaryCompatibilityValidator = "org.jetbrains.kotlinx:binary-compatibility-validator:0.13.2"
 
-ktlint = "com.pinterest:ktlint:0.50.0"
+ktlint = "com.pinterest.ktlint:ktlint:1.0.1"
```
Comment (Goooler) [inline gradle/libs.versions.toml:44]: This should be `"com.pinterest.ktlint:ktlint-cli:1.0.1"` for now, see https://github.com/pinterest/ktlint/releases/tag/1.0.0

Comment (JakeWharton) [reply gradle/libs.versions.toml:44]: Want to send a PR?

Comment (Goooler) [reply gradle/libs.versions.toml:44]: It's #325.

**Pull request merged**